clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -41,6 +41,9 @@
                                                        + "discover the full cluster membership (which may change dynamically), this list need not contain the full set of "
                                                        + "servers (you may want more than one, though, in case a server is down).";
 
+    public static final String CLIENT_DNS_LOOKUP = "client.dns.lookup";
+    public static final String CLIENT_DNS_LOOKUP_DOC = "To enable client DNS lookups on bootstrap.servers";
```

Comment:
We probably want to expand the description a bit. I doubt people would understand what this does without having to check the KIP.

Reply:
Agree with you, not sure how to present best without being to verbose, gave it a try.